src/core/retargeting_engine/python/retargeters/se3_retargeter.py

```
@@ -198,7 +198,7 @@ def output_spec(self) -> RetargeterIOType:
             )
         }
 
-    def compute(self, inputs: RetargeterIO, outputs: RetargeterIO) -> None:
+    def _compute_fn(self, inputs: RetargeterIO, outputs: RetargeterIO, context) -> None:
```

Ruff (0.15.2)

[warning] Unused method argument: `context`

(ARG002)